RPS/src/main/java/GameState.java

```diff
@@ -0,0 +1,40 @@
+public class GameState {
+    private final int noOfRounds;
+    private int humanPoints = 0;
+    private int computerPoints = 0;
+    private int whichRound = 0;
+
+
+    public GameState(int noOfRounds) {
+        this.noOfRounds = noOfRounds;
+    }
+
+    public void addRound(Moves.ValidMoves computer, Moves.ValidMoves human) {
+        if (computer == Moves.ValidMoves.PAPER && human == Moves.ValidMoves.ROCK) { computerPoints++; }
+        if (computer == Moves.ValidMoves.ROCK && human == Moves.ValidMoves.SCISSORS) { computerPoints++; }
+        if (computer == Moves.ValidMoves.SCISSORS && human == Moves.ValidMoves.PAPER) { computerPoints++; }
+        if (computer == Moves.ValidMoves.PAPER && human == Moves.ValidMoves.SCISSORS) { humanPoints++; }
+        if (computer == Moves.ValidMoves.ROCK && human == Moves.ValidMoves.PAPER) { humanPoints++; }
+        if (computer == Moves.ValidMoves.SCISSORS && human == Moves.ValidMoves.ROCK) { humanPoints++; }
```

Comment:
Można by się zastanowić, jak to napisać zgrabniej - zauważ, że każdy `if` ma swoją "odwrotną" wersję, jest więc ich 2 razy więcej, niż mogłoby być. W ramach dodatkowego ćwiczenia możesz przemyśleć, jak to zrobić (podpowiedź: możesz użyć `Map<ValidMoves, ValidMoves>` do przechowywania informacji o zależnościach między figurami).